webserver/__init__.py

```diff
@@ -1,68 +1,83 @@
 import os
 import sys
+
+from flask.config import Config
+
 sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+basepath = os.path.dirname(os.path.realpath(__file__))
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
 import logging
 import logging.handlers as handlers
 
-from helpers.StreckennetzSteffi import StreckennetzSteffi
-from database.engine import DB_CONNECT_STRING
+# Create app with changed paths  https://stackoverflow.com/a/42791810
+app = Flask(__name__, instance_relative_config=True,
+        template_folder='website', static_folder='website/', static_url_path='')
 
-basepath = os.path.dirname(os.path.realpath(__file__))
-
-streckennetz = StreckennetzSteffi(prefer_cache=True)
-from webserver.predictor import Predictor
-pred = Predictor()
-db = SQLAlchemy()
+logHandler = handlers.TimedRotatingFileHandler(basepath + '/logs/website.log', when='midnight', backupCount=100)
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+logHandler.setFormatter(formatter)
+logHandler.setLevel(logging.INFO)
+app.logger.addHandler(logHandler)
+app.logger.setLevel(logging.INFO)
+logger = app.logger
```

Comment:
Please just use `app.logger` and not `logger`

Reply:
Is `logger` used somewhere?

Reply:
No